source/__tests__/integration/import-profile.test.js

```diff
@@ -126,6 +121,7 @@ describe('imports valid profile without included schema url from json file', ()
 
 test('imports valid v0.0.9 profile from json file', async () => {
   expect.assertions(3)
+  await page.$eval('form[name="profileForm"]', e => e.reset())
```

Comment:
Can you say more about this change and the similar ones below? Seeking only to grok better.

Reply:
i'm also curious about this!

Reply:
I did `page.screenshot` to figure out why test was failing, and the image in the failing test was showing the wrong profile.  This was one of the things I tried ... but it's not the one that made it work!  Fixed with new commit arriving imminently.